src/types/types.ts

```diff
@@ -1010,6 +1011,8 @@ export const ErrorStatusCode = {
 
   /** @see https://www.meilisearch.com/docs/reference/errors/error_codes#invalid_facet_search_facet_query */
   INVALID_FACET_SEARCH_FACET_QUERY: 'invalid_facet_search_facet_query',
+
+  INVALID_SEARCH_RANKING_SCORE_THRESHOLD:'invalid_search_ranking_score_threshold',
```

Comment:
Shouldn't we also add `invalid_similar_ranking_score_threshold` based on the usage page (last section): https://meilisearch.notion.site/Filter-by-score-usage-224a183ce7b24ca99b6a9a8da755668a